src/app/shell/personal-cabinet/provider/create-study-subject/create-study-subject.component.spec.ts

```
@@ -0,0 +1,123 @@
+        ChangeDetectorRef
+      ]
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CreateStudySubjectComponent);
+    component = fixture.componentInstance;
+
+    mockStore.select.mockReturnValue(of(null));
+    mockStore.selectSnapshot.mockReturnValue(null);
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)